memcache_proxy/cloudbuild.yaml

```diff
@@ -0,0 +1,21 @@
+# Config for building with Google Container Builder
+#
+# This produces a container with two tags, "latest" and _RC_NAME, which must be
+# specified via a command-line flag.
+#
+# Run with:
+#   gcloud container builds submit --config cloudbuild.yaml . \
+#     --substitutions=_RC_NAME=20180101-RC00
+
+steps:
+  - name: 'gcr.io/cloud-builders/docker'
+    args: ['build', '-f', 'Dockerfile.builder', '-t', 'script-runner', '.']
+  - name: 'script-runner'
+    args: ['./build.sh']
```

Comment:
Yes, it will; I'm not sure what your question is here. If you're worried about the binary being persisted, it is written outside the script-runner container in the `/workspace` directory that Container Builder sets up, so it is able to be used by the build rule in the next step.

Reply:
Got it. Thanks for explaining.